docs/PROFILES.md

```diff
@@ -0,0 +1,28 @@
+# Profiles pages
+
+## Testing
+
+### Firefox
+
+To test the `profiles` app locally on __Firefox__ :
+1. Edit `bridgeConnection.saga` on `l17` 
+```
+const extensionId = '';
+```
+
+2. `yarn start:firefox`
```

Comment:
You don't have to run the extension to launch profiles, right?

Reply:
Not to launch it, but in order for the subscription - Click on subscribe -  to work you do.